### ISSUE TITLE
Consider using require as a statement, not a function.

### ISSUE THREAD

**Issue by yyxx9988** [yiisoft/yii2-app-basic#123]
It's not necessary to write:
```php
<?php
require('somefile.php');
?>
```
The following:
```php
<?php
require 'somefile.php';
?>
```
Is preferred, it will prevent your peers from giving you a hard time and a trivial conversation about what require really is.

see http://php.net/manual/en/function.require.php#75875
see http://php.net/manual/en/function.include.php

**Comment by rob006** [comment]
👍 This also affects core framework. php-cs-fixer has fixer this, so I could adjust https://github.com/yiisoft/yii2/pull/14100 and add rule for this.

**Comment by samdark** [comment]
I'm fine with it.

**Comment by rob006** [comment]
@SilverFire ? It is not yet fixed in this template.

**Comment by SilverFire** [comment]
It's fixed locally, will push soon. Sorry, closed too early

**Comment by cebe** [comment]
@SilverFire as far as I see there is nothing pushed yet. Make a commit with "Fix #123" and it will close when its pushed ;)
